lib/aika/accounts/accounts.ex

```diff
@@ -64,4 +64,9 @@ defmodule Aika.Accounts do
     end
   end
 
+  def create_api_token!(user) do
```

Comment:
How come the bang in the function name here? To me it implies we should be calling the `Repo.update!/1`